packages/compass-components/src/components/feedback-popover.spec.tsx

```diff
@@ -44,6 +45,7 @@ const renderFeedbackPopover = (
 describe('FeedbackPopover', function () {
   afterEach(function () {
     cleanup();
+    sinon.restore();
```

Comment:
Same here, don't see usage of sinon anywhere in the test suite.

Reply:
Removed in https://github.com/mongodb-js/compass/pull/5400/commits/f4cfa4cbdb514b787e640e05488f9606c11c8303, thanks for spotting!